cmd/spire-agent/cli/api/api_test.go

```diff
@@ -100,10 +102,12 @@ func TestFetchJWTCommand(t *testing.T) {
   {
     "svids": [
       {
+		"hint": "external",
```

Comment:
This line is using tabs when the rest of the file uses spaces. Can you changes this to spaces match the rest of the file?